rig/routing_table/__init__.py

```diff
@@ -0,0 +1,4 @@
+from rig.routing_table.entries import RoutingTableEntry, Routes
+from rig.routing_table.exceptions import MinimisationFailedError
+from rig.routing_table.ordered_covering import minimise
```

Comment:
A comment stating that this line is responsible for choosing the "default" minimiser would be good.